device/cluster.cpp

```diff
@@ -1712,14 +1612,22 @@ void* Cluster::host_dma_address(std::uint64_t offset, chip_id_t src_device_id, u
     }
 }
 
-// Wrapper for throwing more helpful exception when not-enabled pci intf is accessed.
+// Wrapper for throwing a more helpful exception when trying to access non pci enabled interface.
 inline TTDevice* Cluster::get_tt_device(chip_id_t device_id) const {
     log_assert(chips_.find(device_id) != chips_.end(), "Device id {} not found in cluster.", device_id);
     auto tt_device = chips_.at(device_id)->get_tt_device();
     log_assert(tt_device != nullptr, "TTDevice not found for device: {}", device_id);
     return tt_device;
 }
 
+// Wrapper for throwing a more helpful exception when trying to access non pci enabled interface.
+inline TLBManager* Cluster::get_tlb_manager(chip_id_t device_id) const {
+    log_assert(chips_.find(device_id) != chips_.end(), "Device id {} not found in cluster.", device_id);
+    auto tlb_manager = chips_.at(device_id)->get_tlb_manager();
```

Comment:
Not sure if this will get optimized to a single lookup.  If it's in a perf path, consider,
```
auto it = chips_.find(device_id); // once
log_assert(...
auto* manager = it->second->get_tlb_manager()
```

Reply:
Applied this to get_tt_device. The get_tlb_manager function has changed